docs/pages/deploy-a-cluster/deployments/aws-terraform.mdx

```diff
@@ -295,12 +295,13 @@ $ terraform import aws_acm_certificate.cert <certificate_arn>
 
 ### Instances
 
-Our reference deployment will provision the following instances for your cluster by default:
+Our reference deployment will provision the following instances for your cluster by default with AWS Auto Scaling Groups (ASG):
```

Comment:
It's no longer clear if the _by default_ refers to ASGs for provisioning instances.

Reply:
Update language to indicate settings.  This also referred to all the settings like server type and count.